Traceback (most recent call last):
  File "C:\Anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "C:\Anaconda3\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Anaconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Anaconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 584, in run_until_complete
    return future.result()
  File "C:\Anaconda3\lib\site-packages\nbclient\client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "C:\Anaconda3\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Anaconda3\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
%matplotlib inline
import numpy as np
import matplotlib.pyplot as plt

x = np.array([0,2,4,6,8])
y = np.array([13,29,35,45,48])

ax.grid('True',ls=':')

x_model = np.linspace(0,10,50)
y_model = x_model*X_hat[0,0] + X_hat[1,0]

fig,ax = plt.subplots()

ax.plot(x,y,'ro');
ax.plot(x_model,y_model,'b',ls=':');

ax.set_xlim(0,10);
ax.set_ylim(0,60);
ax.set_xlabel('Number of years after 2004 ');
ax.set_ylabel('Sales (million dollars)');

ax.grid('True',ls=':')
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
[1;32m<ipython-input-71-ba323c8ebd62>[0m in [0;36m<module>[1;34m[0m
[0;32m      6[0m [0my[0m [1;33m=[0m [0mnp[0m[1;33m.[0m[0marray[0m[1;33m([0m[1;33m[[0m[1;36m13[0m[1;33m,[0m[1;36m29[0m[1;33m,[0m[1;36m35[0m[1;33m,[0m[1;36m45[0m[1;33m,[0m[1;36m48[0m[1;33m][0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      7[0m [1;33m[0m[0m
[1;32m----> 8[1;33m [0max[0m[1;33m.[0m[0mgrid[0m[1;33m([0m[1;34m'True'[0m[1;33m,[0m[0mls[0m[1;33m=[0m[1;34m':'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      9[0m [1;33m[0m[0m
[0;32m     10[0m [0mx_model[0m [1;33m=[0m [0mnp[0m[1;33m.[0m[0mlinspace[0m[1;33m([0m[1;36m0[0m[1;33m,[0m[1;36m10[0m[1;33m,[0m[1;36m50[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;31mNameError[0m: name 'ax' is not defined
NameError: name 'ax' is not defined

